package.json

```diff
@@ -16,64 +16,64 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=18.15.0"
```

Comment:
The `.github/workflows/testPublish.tml` file needs to be configured to execute the testing on the same node versions we are flagging here in the package.json.

Reply:
I see, I will follow [this](https://github.com/AlaskaAirlines/auro-avatar/pull/45/files#diff-3358abb4a7abaf5862dcf8a68e95ee8431a7adf98d0d4c576444fff439dd665aR18) change as reference.

Reply:
Please reference this page in the docSite. https://auro.alaskaair.com/node-support. Please update the `settings.yml` file too.